glotaran/builtin/models/kinetic_image/kinetic_image_matrix.py

```diff
@@ -67,21 +68,11 @@ def kinetic_matrix(
             compartments.append(baseline_compartment)
             matrix = np.concatenate((matrix, baseline), axis=1)
 
-    if isinstance(dataset_descriptor.irf, IrfGaussianCoherentArtifact):
```

Comment:
I assume this is further cleanup resulting from the move of coherent artifact from kinetic_image to kinetic_spectrum class?